week1/homework-week1/Program.cs

```diff
@@ -0,0 +1,47 @@
+﻿// 1. String manipulation
+string input = "word and world";
+string reversed = string.Empty;
+for (int i = input.Length - 1; i >= 0; i--){
+    reversed += input[i];
+}
+Console.WriteLine($"Reversed input value: {reversed}");
+
+//2. String/Math
+string inputVowels = "Intellectualization";
+int len = inputVowels.Length;
+int vowelCount = 0;
+for (int i = 0; i < len; i++){
+ if (inputVowels[i] == 'a'|| inputVowels[i] == 'i' || inputVowels[i] == 'u' || inputVowels[i] == 'e' || inputVowels[i] == 'o' || inputVowels[i] == 'A'|| inputVowels[i] == 'I' || inputVowels[i] == 'U' || inputVowels[i] == 'E' || inputVowels[i] == 'O')
+    {
+ vowelCount++;
+    }

```

Comment:
This is a solution that works good but if you want a small challenge you can look into using an array for the vowels and then check if the string contains any of the elements in the array

Reply:
I will try, Thanks Calin